packages/shared/src/components/cards/CollectionCard/CollectionCard.tsx

```diff
@@ -29,6 +30,7 @@ export const CollectionCard = forwardRef(function CollectionCard(
   }: PostCardProps,
   ref: Ref<HTMLElement>,
 ) {
+  const image = usePostImage(post);
   const clamp = (() => {
     if (post.image) {
```

Comment:
Should this not check on `image` then?

Reply:
Right, forgot to update the other one. Maybe I should introduce a function that generates this since they are of the same result.